tests/FSharp.Compiler.ComponentTests/ErrorMessages/ExtendedDiagnosticDataTests.fs

```diff
@@ -180,13 +181,53 @@ let f2 (x: inref<'T>) = f1 &x
 """
     |> typecheckResults
     |> checkDiagnosticData
-       (1, "Type mismatch. Expecting a\n    'outref<'T>'    \nbut given a\n    'inref<'T>'    \nThe type 'ByRefKinds.Out' does not match the type 'ByRefKinds.In'")
+       (1, "Type mismatch. Expecting a\n    'outref<'T>'    \nbut given a\n    'inref<'T>'    \nThe type 'ByRefKinds.In' does not match the type 'ByRefKinds.Out'")
```

Comment:
<img width="2146" height="647" alt="Image" src="https://github.com/user-attachments/assets/1c8a8e9b-8058-41bf-96ba-1df79c256b20" />

In this case (and other with a difference in generic parameters) the main message is ok, but its clarification at the end has the same problem with the order.